### DISCUSSION TITLE
Handle Long Exit Codes

### DISCUSSION THREAD

**Discussion by dbuechel** [wixtoolset/issues#6851] [Questions] [answered]
We'd like to handle the exit code of an `ExePackage` in a setup bundle. However, the exit code to handle is bigger than an integer (if I haven't missed anything obvious). I couldn't find any hints online, thus I'd like to ask whether it indeed isn't possible to handle exit codes which are bigger than an integer?

The exit code in question originates from MSFTs WebView2 bootstrapper, namely value `0x8004090d` which we'd like to handle gracefully (see https://github.com/SafeExamBrowser/seb-win-refactoring/issues/429).

**Comment by rseanhall** [comment]
This is fixed in v4: https://github.com/wixtoolset/issues/issues/4176.

**Reply by dbuechel** [reply]
Excellent, thanks for the quick response!

**Comment by rseanhall** [comment, marked as answer]
It turns out this was fixed in v3.9: https://github.com/wixtoolset/issues/issues/4319. `0x8004090D` should be able to be specified as a signed integer: `-2147219187`.

**Reply by dbuechel** [reply]
That's amazing, thanks a lot for the update! Then it was my mistake after all, as I didn't convert the hex code to a signed integer... 😬
